service/matching/reachability.go

```diff
@@ -129,56 +130,26 @@ func (rc *reachabilityCalculator) run(ctx context.Context, buildId string) (enum
 
 // getBuildIdsOfInterest returns a list of build ids that point to the given buildId in the graph
 // of redirect rules and adds the given build id to that list.
-// It considers rules if the deletion time is within versioningReachabilityDeletedRuleInclusionPeriod.
+// It considers rules if the deletion time is nil or within the given deletedRuleInclusionPeriod.
 func (rc *reachabilityCalculator) getBuildIdsOfInterest(
 	buildId string,
-	includeRecentlyDeleted bool) []string {
-	return append(rc.getUpstreamHelper(buildId, includeRecentlyDeleted, nil), buildId)
-}
-
-func (rc *reachabilityCalculator) getUpstreamHelper(
-	buildId string,
-	includeRecentlyDeleted bool,
-	visited []string) []string {
-	var upstream []string
-	visited = append(visited, buildId)
-	directSources := rc.getSourcesForTarget(buildId, includeRecentlyDeleted)
+	deletedRuleInclusionPeriod *time.Duration) []string {
```

Comment:
Please put the closing paren and return value on a line of their own, otherwise it's difficult to find them when writing a multi-line signature